domains/authorization/Authorization.API/API/ValueObjects/Tin.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+
+namespace API.ValueObjects;
+
+public readonly record struct Tin
```

Comment:
Make ValueObject base object instead of record and inherrit from that

Reply:
Do we have any base configuration we can throw in the Base Value object?

Reply:
I don't understand the question

Reply:
I added a Base ValueObject